### PR TITLE
Pass by reference

### PR DESCRIPTION
**Pull request by eggrobin**
More #718.

### CODE REVIEW

File: base/pull_serializer_test.cpp [hunk 1]
```diff
@@ -65,10 +68,9 @@ class PullSerializerTest : public ::testing::Test {
   // Returns the first string in the list.  Note that the very first string is
   // always discarded.
   Bytes OnFull(Bytes const bytes,
-               not_null<std::list<std::string>*> const strings) {
-    strings->push_back(
-        std::string(reinterpret_cast<const char*>(&bytes.data[0]),
-                    static_cast<size_t>(bytes.size)));
+               std::list<std::string>& strings) {
```
Comment (pleroy) [inline base/pull_serializer_test.cpp]: Would fit on the previous line.

File: ksp_plugin_test/plugin_test.cpp [hunk 2]
```diff
@@ -284,10 +284,10 @@ class PluginTest : public testing::Test {
   // Inserts a vessel with the given |guid| and makes it a satellite of Earth
   // with relative position |satellite_initial_displacement_| and velocity
   // |satellite_initial_velocity_|.  The vessel must not already be present.
-  // Increments the counter |*number_of_new_vessels|.  |number_of_new_vessels|
+  // Increments the counter |number_of_new_vessels|.  |number_of_new_vessels|
   // must not be null.
```
Comment (pleroy) [inline ksp_plugin_test/plugin_test.cpp]: Fix the comment.

File: physics/ephemeris_test.cpp [hunk 3]
```diff
@@ -130,25 +128,24 @@ class EphemerisTest : public testing::Test {
                                         4e8 * Metre,
                                         0 * Metre});
     Length const semi_major_axis = (q1 - q2).Norm();
-    *period = 2 * π * Sqrt(Pow<3>(semi_major_axis) /
-                               (earth->gravitational_parameter() +
-                                moon->gravitational_parameter()));
-    *centre_of_mass =
-        Barycentre<Position<ICRFJ2000Equator>, Mass>(
-            {q1, q2}, {earth->mass(), moon->mass()});
```
Comment (pleroy) [inline physics/ephemeris_test.cpp]: I preferred this formatting.

File: base/pull_serializer_test.cpp (hunk 1 above)
Comment (eggrobin) [inline base/pull_serializer_test.cpp]: Done.

File: ksp_plugin_test/plugin_test.cpp (hunk 2 above)
Comment (eggrobin) [inline ksp_plugin_test/plugin_test.cpp]: Done.

File: physics/ephemeris_test.cpp (hunk 3 above)
Comment (eggrobin) [inline physics/ephemeris_test.cpp]: Done.

**Pull request merged**